contracts/sei-tester/src/contract.rs

```diff
@@ -244,6 +247,32 @@ pub fn process_finalize_block(
     Ok(response)
 }
 
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn reply(_deps: DepsMut<SeiQueryWrapper>, _env: Env, msg: Reply) -> Result<Response, StdError> {
+    match msg.id {
+        PLACE_ORDER_REPLY_ID => handle_place_order_reply(msg),
```

Comment:
a qq here, the reply for a submsg is FIFO correct? so what's the best practice to know which reply is for which submsg here, e.g. in action #3 and #4 I place order #3 and #4, how do i match them in the reply? do i want to use an ever-increasing counter here?

Reply:
Yes they are FIFO. This case would be more complicated. I believe the way to handle this would be to create a state variable that does what you say - has a counter and in the reply load the state bar to see what the counter is set to.